prowler/lib/check/check.py

```diff
@@ -110,6 +111,7 @@ def parse_checks_from_file(input_file: str, provider: str) -> set:
     checks_to_execute = set()
     f = open_file(input_file)
     json_file = parse_json_file(f)
+    f.close()
```

Comment:
I prefer if you use the following with the context manager:
```python
with open("hello.txt") as my_file:
    print(my_file.read())
```

Reply:
Done